src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -79,101 +87,41 @@ public static Colony createColony(World w, BlockPos pos, EntityPlayer player)
     }
 
     /**
-     * Get Colony by UUID
-     *
-     * @param id    ID of colony
-     * @return Colony with given ID
+     * Syncs the achievements for all colonies.
      */
-    public static Colony getColony(int id)
-    {
-        return colonies.get(id);
-    }
-
-    /**
-     * Get colony that contains a given coordinate
-     *
-     * @param w     World
-     * @param pos   coordinates
-     * @return      Colony at the given location
-     */
-    public static Colony getColony(World w, BlockPos pos)
-    {
-        List<Colony> coloniesInWorld = coloniesByWorld.get(w.provider.getDimensionId());
-        if (coloniesInWorld == null)
-        {
-            return null;
-        }
-
-        for (Colony c : coloniesInWorld)
-        {
-            if (c.isCoordInColony(w, pos))
-            {
-                return c;
-            }
-        }
-
-        return null;
+    public static void syncAllColoniesAchievements(){
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)